src/playground-ide.ts

```diff
@@ -140,12 +140,32 @@ export class PlaygroundIde extends LitElement {
   @property({attribute: 'project-src'})
   projectSrc?: string;
 
+  /**
+   * Base URL for script execution sandbox.
+   *
+   * It is highly advised to change this property to a URL on a separate origin
+   * which has no privileges to perform sensitive actions or access sensitive
+   * data. This is because this element will execute arbitrary JavaScript, and
+   * does not have the ability to sanitize or sandbox it.
+   *
+   * This URL must host the following files from the playground-elements
+   * package:
+   *   1. playground-service-worker.js
+   *   2. playground-service-worker-proxy.html
```

Comment:
Do you think it'd be easier to follow if the attribute pointed directly to the SW-registering HTML file?

Reply:
As written, you can just point it at a URL that is hosting `node_modules/playground-elements`, and don't have to know what the name of the files should be. This documentation is here just so people know what needs to be there when they are building/deploying. Seems simpler this way? Not sure why somebody would want to change the HTML file name.